enhancements/hypershift/service-publishing-api-evolution.md

```
@@ -0,0 +1,1487 @@
+message: "spec.services is deprecated;
+  use spec.publishing"
+```
+
+  This rule:
+  - **Create with `spec.services[]`:** rejected.
+  - **Update without changing `spec.services[]`:**
+    ratcheted (passes).
+  - **Update modifying `spec.services[]`:** rejected
+    (field changed, ratcheting does not apply).
+
+- HO translation path remains active.
```

markdownlint-cli2 (0.23.2)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 0

(MD005, list-indent)